packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -103,11 +103,22 @@ function testFixtures(options?: RollupLwcOptions) {
             pattern: '**/index.js',
         },
         async ({ filename, dirname, config }) => {
-            const compiledFixturePath = await compileFixture({
-                input: filename,
-                dirname,
-                options,
-            });
+            let compiledFixturePath;
+
+            try {
+                compiledFixturePath = await compileFixture({
+                    input: filename,
+                    dirname,
+                    options,
+                });
+            } catch (err: any) {
+                // Filter out the stacktrace, just include the LWC error message
+                const message = err?.message?.match(/(LWC\d+[^\n]+)/)?.[1];
+                return {
+                    'expected.html': '',
+                    'error.txt': message ?? '',
+                };
+            }
```

Comment:
I figure we might as well start testing compiler errors in `engine-server`. It's convenient for SSR tests since they effectively do double-duty as a compiler and runtime.